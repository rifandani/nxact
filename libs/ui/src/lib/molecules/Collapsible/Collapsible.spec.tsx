import renderer from 'react-test-renderer';
import {
  Collapsible,
  CollapsibleContent,
  CollapsibleTrigger,
} from './Collapsible';

describe('Collapsible component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Collapsible>
          <CollapsibleTrigger>Triggerer</CollapsibleTrigger>
          <CollapsibleContent>Collapsible Content</CollapsibleContent>
        </Collapsible>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
