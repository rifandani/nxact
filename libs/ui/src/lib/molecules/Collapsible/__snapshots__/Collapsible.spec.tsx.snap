// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button regular component should render correctly 1`] = `
<div
  data-state="closed"
>
  <button
    aria-controls="radix-:r0:"
    aria-expanded={false}
    data-state="closed"
    onClick={[Function]}
    type="button"
  >
    Triggerer
  </button>
  <div
    data-state="closed"
    hidden={true}
    id="radix-:r0:"
    style={
      Object {
        "--radix-collapsible-content-height": undefined,
        "--radix-collapsible-content-width": undefined,
      }
    }
  />
</div>
`;

exports[`Collapsible component should render correctly 1`] = `
<div
  data-state="closed"
>
  <button
    aria-controls="radix-:r0:"
    aria-expanded={false}
    data-state="closed"
    onClick={[Function]}
    type="button"
  >
    Triggerer
  </button>
  <div
    data-state="closed"
    hidden={true}
    id="radix-:r0:"
    style={
      Object {
        "--radix-collapsible-content-height": undefined,
        "--radix-collapsible-content-width": undefined,
      }
    }
  />
</div>
`;
